core/includes/color.inc

```diff
@@ -47,7 +47,7 @@ class Color {
     // Ignore '#' prefixes.
     $hex = ltrim($hex, '#');
 
-    // Convert shorhands like '#abc' to '#aabbcc'.
```

Comment:
I think the plural is more correct, in this case. I would rather put a comma after *shorthands* and use *for example* instead of *like*.

Reply:
I'm not sure I like the word 'shorthands' in this context - doesn't seem right to me. But I see your point, so I've made it 'Convert shorthand colors' instead. I agree with your other suggestions.